### PR TITLE
Try setting up gitpod for building for each PR to ease reviewing

### PR DESCRIPTION
**Pull request by jywarren**
Building on example at https://github.com/publiclab/image-sequencer/pull/1668

cc @sashadev-sky take a look - gitpod can build a full hosted version for each PR!

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+tasks:
+  - init: npm i && npm run build
+    command: npm run serve:dev
```
Comment (jywarren) [inline .gitpod.yml:3]: Looking at https://github.com/publiclab/Leaflet.DistortableImage/blob/main/package.json, to configure

**Comment by jywarren** [comment]
Hmm!  Test error, probably due to setup, and Travis I think caught it too. But, it did seem to be trying to serve the app in GitPod! So if we can get the settings right perhaps this will work!

```
✔ 0 tests completed
PhantomJS 2.1.1 (Linux x86_64) ERROR
  ReferenceError: Can't find variable: L
  at /workspace/Leaflet.DistortableImage/src/util/DomUtil.js:1:2 <- src/util/DomUtil.es5.js:1:4011
```

**Comment by jywarren** [comment]
Hmm. Still seeing the timeout issue and have documented it in https://github.com/publiclab/Leaflet.DistortableImage/issues/681. If we can resolve that, i believe this will start working!

**Comment by sashadev-sky** [comment]
@jywarren i followed the PR trail and saw we never resolved this. Restarting the build one or two (maybe 3?) times results in a pass so for now I guess that's what we'd have to do. We can merge this if it's ready!

**Comment by sashadev-sky** [comment]
Merging as tests are passing :) @jywarren

**Comment by jywarren** [comment]
This is great, Sasha!!!!! Now it can be tested interactively on the fly in each PR. Thanks!!!

**Pull request merged**